changelogs/fragments/378.yml

```
@@ -0,0 +1,7 @@
+---
+trivial:
+  - node_exporter - update build dependencies for node-exporter-extras package
+  - postgres_exporter - minimum version 0.13.2, maximum 0.15
+  - prometheus - minimum version 2.38, maximum 2.47
+  - alertmanager - minimum version 0.23, maximum 0.26
+  - grafana - minimum version 9.5, maximum 9.5
```

```suggestion
minor_changes:
  - postgres_exporter - minimum version 0.13.2, maximum 0.15.x
  - prometheus - minimum version 2.38, maximum 2.47.x
  - alertmanager - minimum version 0.23, maximum 0.26.x
  - grafana - minimum version 9.5, maximum 9.9.x
```
trivial isn't shown in the generated changelog, and version bumps, while minor, are notable enough they should be shown to the user